src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -1412,6 +1412,13 @@ def load_arguments(self, _):
         c.argument('ppg_type', options_list=['--type', '-t'], help="The type of the proximity placement group. Allowed values: Standard.")
         c.argument('tags', tags_type)
 
+    with self.argument_context('ppg create', min_api='2021-11-01') as c:
+        c.argument('zone', nargs='*', help="Specifies the availability zone where virtual machine, virtual machine scale set or availability set associated with the proximity placement group can be created. Accepted values 1,2,3.")
+
+    for scope in ['ppg create', 'ppg update']:
+        with self.argument_context(scope, min_api='2021-11-01') as c:
+            c.argument('intentvmsizes', nargs='*', help="Specifies possible sizes of virtual machines that can be created in the proximity placement group.")
```

Comment:
It is suggested to mark the configuration of `min_api` at a finer level
```suggestion
        with self.argument_context(scope) as c:
            c.argument('intentvmsizes', nargs='*', min_api='2021-11-01', help="Specifies possible sizes of virtual machines that can be created in the proximity placement group.")
```